src/main/java/Model/Client.java

```diff
@@ -3,28 +3,27 @@
 
 import javax.persistence.*;
 import java.time.LocalDate;
-import java.util.Date;
 
 @Entity
 
 
 public class Client {
-@Id
-@GeneratedValue(strategy = GenerationType.AUTO)
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
     private String name;
-    private String surname;
+    private String lastName;
     private LocalDate dateOfBirth;
     private Integer phoneNumber;
```

Comment:
Kurcze musiałem przeoczyć te 2 rzeczy przy review:
1. Może lepiej ustawić nr telefonu jako string? Pozwoli to na zapisywanie kierunkowych typu +48 ...

Reply:
Myślę, że na ten moment zostawimy to na zasadzie tylko polskie numery (input ograniczony do 9 cyfr), ale jak najbardziej do rozbudowy później.